x-pack/platform/test/alerting_api_integration/security_and_spaces/group3/tests/alerting/bulk_disable.ts

```diff
@@ -187,64 +186,6 @@ export default ({ getService }: FtrProviderContext) => {
           }
         });
 
-        it('should handle disable alert request appropriately when consumer is not the producer', async () => {
```

Comment:
This test is not relevant anymore. This https://github.com/elastic/kibana/pull/183756 removed producer authorization.